timesketch/frontend/src/components/Common/DynamicFormDatetimeInput.vue

```diff
@@ -28,7 +28,7 @@ export default {
   props: ['placeholder', 'label', 'name', 'value', 'display'],
   methods: {
     dateFormatter(dt) {
-      let dateString = dt.toISOString().replace('.000Z', '')
+      let dateString = new Date(dt.getTime() - dt.getTimezoneOffset() * 60000).toISOString().replace('.000Z', '')
```

Comment:
Can we just add a comment about what calculation is going on here? Otherwise LGTM

Reply:
Done.